hvplot/converter.py

```diff
@@ -2566,6 +2566,7 @@ def labels(self, x=None, y=None, data=None):
         self._error_if_unavailable('labels')
         self.use_index = False
         data, x, y = self._process_chart_args(data, x, y, single_y=True)
+        data = data.copy()
```

Comment:
Is there a HoloViews-y way to avoid making a copy? That would be the first call to `copy()` in the converter.

If not, I'd suggest making the copy only in the `elif text not in data.columns` block.

Reply:
Not entirely sure; I just put it in the block.